certified-connectors/WithoutWire Inventory Platform/apiDefinition.swagger.json

```diff
@@ -1116,6 +1124,4455 @@
+                          "description": "A unique value that represents an item",
+                          "title": "Item Number"
+                        },
+                        "CoreItemType": {
+                          "type": "string",
+                          "description": "A string value that indicates whether the item is lot tracked, serial tracked, expiration date tracked, or a standard item.",
+                          "title": "Core Item Type",
+                          "enum": [
+                            "BASIC",
+                            "LOT",
+                            "SERIAL",
+                            "or DATE"
+                          ]
+                        },
+                        "QuantityOrdered": {
+                          "type": "number",
+                          "format": "float",
+                          "description": "A decimal value representing the quantity to pick for the line item"
+                        },
+                        "UomDesc": {
+                          "type": "string",
+                          "description": "A string value that uniquely identifies the unit of measurement."
+                        },
+                        "BaseConversionFactor": {
+                          "type": "number",
+                          "format": "float",
+                          "description": "A decimal value representing the conversion from this unit of measure to the base unit of measure"
+                        },
+                        "SignificantDigits": {
+                          "type": "integer",
+                          "format": "int32",
+                          "description": "An integer value representing the number of decimal places to track for this unit of measure"
+                        },
+                        "AllocationSetName": {
+                          "type": "string",
+                          "description": "The allocation set that the associated inventory record is assigned to.",
+                          "title": "Allocation Set Name"
+                        },
+                        "LineItemComments": {
+                          "type": "array",
+                          "items": {
+                            "type": "string"
+                          },
+                          "description": "A collection of strings to associate with this line on the purchase order."
+                        }
+                      }
+                    },
+                    "description": "A collection of Purchase Order Line Object to be associated with a purchase order.  Receipts are found within the purchase order line object."
+                  }
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/integration/warehouse": {
+      "put": {
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {}
```

Comment:
@PavloNikulinEnavate Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.